app/controllers/pageflow/entries_controller.rb

```diff
@@ -19,9 +19,23 @@ class EntriesController < Pageflow::ApplicationController
     helper RenderJsonHelper
 
     def index
-      theming = Theming.for_request(request).with_home_url.first!
+      respond_to do |format|
+        format.html do
+          @theming = Theming.for_request(request).with_home_url.first!
+          redirect_to(@theming.home_url)
+        end
 
-      redirect_to(theming.home_url)
+        format.atom do
+          @theming = Theming.for_request(request).first
+
+          if @theming
+            @account = @theming.account
+            @entries = EntryFeed::Scope.new(@account.entries).entries
+          else
+            @entries = EntryFeed::Scope.new(Entry).entries
```

Comment:
This makes me feel a bit uneasy. For example, in the case of Hosted Pageflow, this would mean that a customer suddenly gets a feed with everybody's entries soon as a he changes the CNAME of his account, but still has the old domain pointing to our server. Is there really a use case for this global feed?

Reply:
Yes, ours. 😄 

I see  your point, though. I suggest this atom-feed feature will only work on accounts that have a CNAME, agreed? I'll drop the global feed, and implement this in the host app.